conda-forge.yml

```diff
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+# - only affects README installation instructions
+# - edit recipe/conda_build_config.yaml to affect build
+channels:
+  targets:
+    - ["tiledb", "main"]
+github:
+  user_or_org: TileDB-Inc
+  branch_name: master
```

Comment:
The primary branch name for this feedstock repo is "main". It is only older feedstock repos like tiledb-vcf-feedstock that have a default branch of "master"